src/utils/ExtensionUtilities.js

```diff
@@ -58,7 +149,38 @@ export function acceleratedRaycast( raycaster, intersects ) {
 
 }
 
-export function computeBoundsTree( options ) {
+export function computeBoundsTree( options = {} ) {
+
+	if ( this.isBatchedMesh ) {
+
+		if ( ! IS_REVISION_166 ) {
+
+			console.error( 'Three r166+ is required.' );
+			return;
+
+		}
+
+		options = {
+			...options,
+			indirect: false,
```

Comment:
We may want to add a warning that we're overwriting this if a user tries to set indirect to true.